tests/end_to_end/test_docker_monitoring/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Generate events in the manager
+  hosts: wazuh-manager
+  tasks:
+
+    - name: Truncate alerts file
+      shell: echo "" > /var/ossec/logs/alerts/alerts.json
+      become: True
+
+    - name: {{ event_description }}
+      become: True
+      shell: {{ command }}
```

Comment:
```suggestion
    - name: "{{ event_description }}"
      become: True
      shell: "{{ command }}"
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2948/commits/de272050ca3415a878861a8ee0461883c3936f17